graph2tac/loader/data_server.py

```diff
@@ -307,7 +307,7 @@ def __init__(self, proportions : list[int], random_seed : int):
         self.random_seed = random_seed
     def lemma(self, d : Definition) -> int:
         # to make it identical to vasily's loader
-        ident_64 = np.uint64(d.node.identity)
```

Comment:
I think we could actually drop this conversion :-), (while making sure that tests pass) The only purpose of it was to test compatibility with the original loader.

Reply:
I'm fine with dropping it, but lets do it in it's own PR since it will change all the test results (since it changes the train-test splits).  Since I'm changing the format of the result files, I'd like to at least keep the contents the same so I know I'm not breaking stuff as I go.